File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q15.js
import { question } from "readline-sync";
 function main(){
   //Input
   const base = parseFloat(question("Type the base of the triangle: "))
   const height = parseFloat(question("Type the height of the triangle: "))

   //Processing
   const area = (base * height)/2

   //Output
   console.log(`The area of the triangule with base of ${base} and height of ${height} is equal to: ${area} a.u`)
 }
 main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q15.py
def main():
    #Input
    triangle_height = float(input("Enter the triangle height: "))
    triangle_base = float(input("Enter the triangle base: "))

    #Processing
    area = (triangle_base * triangle_height)/2

    #Output
    print(f"The area of the triangle with {triangle_height} of height and {triangle_base} of base is equal to: {area} a.u")


main()